tests/i18n/test_extraction.py

```diff
@@ -235,6 +251,11 @@ def test_template_message_context_extractor(self):
             self.assertIn('msgctxt "Special trans context #3"', po_contents)
             self.assertMsgId("Translatable literal #7c", po_contents)
 
+            # {% trans %} with a filter
+            for code in 'abcdefgh':
```

Comment:
In the context of this commit it's understandable what "abcdefgh" means, but describing the purpose in a comment would be helpful to future readers looking at the code outside the context of this commit.